android/build.gradle

```diff
@@ -6,12 +6,15 @@ ext {
 }
 
 buildscript {
+    ext {
+        agp_version = '8.1.3'
```

Comment:
Is there a specific reason for downgrading the gradle build tools?

Reply:
rollback to gradle:8.2.2 done!